.gitignore

```diff
@@ -0,0 +1 @@
+*.yml
```

Comment:
if you are ignoring this file, then it would be nice to rename `config.yml` file to something like `config.yml.example`